### DISCUSSION TITLE
news/2023-06-02-user-forum-11

### DISCUSSION THREAD

**Discussion by giscus** [mhm-ufz/mhm-ufz.github.io#11] [website]
# news/2023-06-02-user-forum-11

The 11th User-Forum on June 7th 2023 in Zoom…

https://mhm-ufz.org/news/2023-06-02-user-forum-11

**Comment by sadiyanti** [comment]
Hi Sebastian, i missed the talk.. was it recorded? I am in particular interested in applying optional data of groundwater head measurements into the model and how to set this on the mhm.nml 
Thanks,
Yanti - Western Australia
